portfolio/src/main/webapp/script.js

```diff
@@ -75,34 +75,34 @@ function addRandomBibleVerse() {
 }
 
 /**
- * Changes Navbar style depending on scroll position
+ * Changes Navbar style depending on scroll position.
  */
 window.addEventListener('scroll', function() {
   if (document.body.scrollTop > 80 || 
       document.documentElement.scrollTop > 80) {
     document.getElementById('navbar').style.padding = '30px 10px';
     document.getElementById('navbar').style.backgroundColor = 
         'rgba(0, 0, 0, 0.25)';
-    document.getElementById('logo').style.fontSize = '25px';
+    document.getElementById('myName').style.fontSize = '25px';
   }
   else {
     document.getElementById('navbar').style.padding = '80px 10px';
     document.getElementById('navbar').style.backgroundColor = 
         'rgba(0, 0, 0, 1)';
-    document.getElementById('logo').style.fontSize = '35px';
+    document.getElementById('myName').style.fontSize = '35px';
   }
 })
 
 /**
- * Fetches a message from the server and adds it to the DOM.
+ * Fetches all messages submitted via the form from the server and adds it to the DOM.
```

Comment:
nit: it sounds like the form is in the server - maybe 'all messages (submitted via the form) from the server...'

Reply:
Done!